.github/lightning-probot.yml

```diff
@@ -0,0 +1 @@
+tracking_issue: 10530
```

Comment:
Tested in my fork. Seems to be working

But this file needs to be merged to master.

Reply:
https://github.com/carmocca/pytorch-lightning/issues/4 :tada: